.github/workflows/plugin_install.yml

```diff
@@ -0,0 +1,70 @@
+name: Plugin Install
+
+on: [push, pull_request, workflow_dispatch]
+
+env:
+  OPENSEARCH_VERSION: "1.4.0"
```

Comment:
OpenSearch 1.4.0 dependency is only on `1.x` branch. How did this pass?

Reply:
This workflow downloads the min tar.gz of opensearch, and it runs with it.  In a follow up PR I've unified these version identifiers into one place

Reply:
Ok. But the OpenSearch version on `main`branch shouldn't be `1.4.0` here. Will this be fixed in the follow up PR?

Reply:
This will be updated when we can switch to the 2.0.0 builds of OpenSearch, until then we will be pointed to the 1.4.0 to get past jar hell issues - see https://github.com/opensearch-project/OpenSearch/pull/2548